stylus/ui-components/modals.styl

```diff
@@ -52,17 +52,23 @@ $modals
         // hide the border-radius
         overflow          hidden
 
-        .coz-modal-content,
-        .coz-modal-section
+        .coz-modal-content
             padding 0 1.5em
 
         .coz-modal-section
+            padding 1.5em
+
+        .coz-modal-section
             border-top 1px solid grey-09
 
+        .coz-modal-content:first-of-type
```

Comment:
What do you suggest ? Use some kind of `coz-modal-content--first` class ? It's a pain to implement and always complicate code.
Do you have some examples of maintenance issues with `:first-of-type` ?